research/flamby/fed_heart_disease/fedper/client.py

```diff
@@ -0,0 +1,141 @@
+import argparse
+import os
+from logging import INFO
+from pathlib import Path
+from typing import Optional, Sequence, Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+from flamby.datasets.fed_heart_disease import BATCH_SIZE, LR, NUM_CLIENTS, BaselineLoss
+from flwr.common.logger import log
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from fl4health.checkpointing.checkpointer import BestMetricTorchCheckpointer, TorchCheckpointer
+from fl4health.clients.moon_client import MoonClient
+from fl4health.utils.losses import LossMeterType
+from fl4health.utils.metrics import Accuracy, Metric
+from fl4health.utils.random import set_all_random_seeds
+from research.flamby.fed_heart_disease.fedper.fedper_model import FedHeartDiseaseFedPerModel
+from research.flamby.flamby_data_utils import construct_fed_heard_disease_train_val_datasets
+
+
+class FedHeartDiseaseFedPerClient(MoonClient):
+    def __init__(
+        self,
+        data_path: Path,
+        metrics: Sequence[Metric],
+        device: torch.device,
+        client_number: int,
+        learning_rate: float,
+        loss_meter_type: LossMeterType = LossMeterType.AVERAGE,
+        checkpointer: Optional[TorchCheckpointer] = None,
+    ) -> None:
+        super().__init__(
```

Comment:
Maybe add a comment here that states that the Moon loss weight defaults to zero?